src/GraphQL/Conversion/ListConverterFactories/ArrayListConverterFactory.cs

```diff
@@ -0,0 +1,24 @@
+namespace GraphQL.Conversion;
+
+internal sealed class ArrayListConverterFactory : ListConverterFactoryBase
```

Comment:
All the default factories are internal.  If someone wants a reference they can call `ValueConverter.GetListConverterFactory(typeof(int[]))` or similar.  I don't see why anyone would need a reference besides accessing it through `ValueConverter.GetListConverterFactory`.